pkgs/development/compilers/graalvm/community-edition/default.nix

```diff
@@ -20,8 +46,33 @@ in
   # directory"/tmp/SVM-4194439592488143713"): error=0, Failed to exec spawn
   # helper: pid: 19865, exit value: 1"
   graalvm17-ce = mkGraal rec {
+    config = {
+      x86_64-darwin = {
+        arch = "darwin-amd64";
+        products = commonProducts ++ [ "python-installable-svm" ];
+      };
+      x86_64-linux = {
+        arch = "linux-amd64";
+        products = commonProducts ++ [ "python-installable-svm" ];
+      };
+    };
     version = "22.0.0.2";
     javaVersion = "17";
-    platforms = [ "x86_64-linux" "x86_64-darwin" ];
+    platforms = builtins.attrNames config;
+  };
+
+  graalvm17-ce-dev = mkGraal rec {
```

Comment:
What is the idea when `graalvm17` final is out to `darwin-aarch64`, to alias `graalvm-17-ce-dev = `graalvm17-ce`?

Ideally I would prefer that we don't add a new package and if possible it would be better to have `graalvm17-ce` for  `darwin-aarch64` to map to `dev` versions temporary. But if this is not possible I am fine with the solution above too.

Reply:
Aliasing would work.

If we decide to drop the env builds we just remove the `graalvm17-ce-dev` derivation again and add the `aarch64-darwin` platform to:
```nix
graalvm17-ce = mkGraal rec {
    config = {
      x86_64-darwin = {
        arch = "darwin-amd64";
        products = commonProducts ++ [ "python-installable-svm" ];
      };
      x86_64-linux = {
        arch = "linux-amd64";
        products = commonProducts ++ [ "python-installable-svm" ];
      };
      aarch64-darwin = {
        arch = "darwin-aarch64";
        products = [ "supported products on darwin-aarch64 on release" ];
      };
    };
    version = "22.0.0.2";
    javaVersion = "17";
    platforms = builtins.attrNames config;
  };
```

Reply:
We can never remove a name though, this is why aliasing would be necessary in this case.

Reply:
@thiagokokada You are right. We would have to add something like this to `pkgs/top-level/aliases.nix`:
```nix
graalvm17-ce-dev = throw "graalvm17 dev builds were discontinued in ..."
```
This is a bit ugly.

The next graalvm release will be on April 26, 2022: 22.1.0, 21.3.2. Maybe it makes sense 
> to have graalvm17-ce for darwin-aarch64 to map to dev versions temporary.

until then. I'll have a look at it tomorrow.